processor/src/range/aux_trace.rs

```diff
@@ -156,7 +89,7 @@ impl AuxTraceBuilder {
         let mut rc_user_op_idx = 0;
 
         // the first half of the trace only includes values from the operations.
-        for (clk, range_checks) in self.cycle_range_checks.range(0..=self.start_16bit as u32) {
+        for (clk, range_checks) in self.cycle_range_checks.range(0..self.start_16bit as u32) {
```

Comment:
> The name start_16bit is confusing because technically, we have only the 16-bit segment now. I guess what we are describing there is the end of the padding section. Should it be padding_end?

Sounds good. I'll change all similar instances with a name to similar effect.

> Should we rename p1 into p0 or maybe even b_range?

I think maybe `b_range` is clearer since it's more descriptive. If we are choosing between `p0` and `p1` I think for other components of the trace, the product column naming is one-indexed (for decoder and hasher) so I didn't change it.

Reply:
Actually, even `padding_end` might be confusing since it's unclear if it means the last padding row or first values row and refactoring the code to fit the name seems worse. Would something like `values_start` or `start_values` or `values_start_row` be better?